File: code/cfg/NamesCfg.cs
using System;
namespace cfg{
public class NamesCfg  {
public const int __TYPE_ID__ = 1000003;
public int GetTypeid() { return __TYPE_ID__; }
public readonly System.Collections.Generic.Dictionary<int, string> names = new System.Collections.Generic.Dictionary<int, string>();
public NamesCfg(CSVStream fs) {
while(!fs.IsSectionEnd()) {
this.names[fs.GetInt()] = fs.GetString();
}
}
}
}

File: code/cfg/NamesCfg.java
package cfg;
public class NamesCfg  {
public final static int __TYPE_ID__ = 1000003;
public final int getTypeid() { return __TYPE_ID__; }
public final java.util.Map<Integer, String> names = new java.util.HashMap<Integer, String>();
public NamesCfg(CSVStream fs) {
while(!fs.isSectionEnd()) {
this.names.put(fs.getInt(), fs.getString());
}
}
}